trace/config.go

```diff
@@ -24,12 +24,14 @@ type Config struct {
 	DefaultSampler Sampler
 }
 
-// SetConfig sets the global tracing configuration.
-func SetConfig(cfg Config) {
+// ApplyConfig applies changes to the global tracing configuration.
+//
+// Fields not provided in the given config is going to be preserved.
```

Comment:
Minor typo:
s/is going to/are going to/

Reply:
Done